src/App.js

```diff
@@ -1,19 +1,43 @@
 import React from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+import { useState } from 'react';
 
 const App = () => {
+  // const [likes, setLikes] = useState(0);
```

Comment:
This actually would have been great to have! That way, you could update the number of likes in the `updateChatData` function every time a button is clicked rather than needing to go through the whole list of message to count the number of liked messages.